qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/kubernetes.tf

```diff
@@ -102,6 +102,13 @@ module "kubernetes-conda-store-server" {
 
   name              = "qhub"
   namespace         = var.environment
+
+  external-url = var.endpoint
+
+{% if cookiecutter.certificate.type == "existing" %}
+  certificate-secret-name = "{{ cookiecutter.certificate.secret_name }}"
```

Comment:
This will follow the same approach we did for clearml right? when using a provisioned certificate